source/Service/MessageService.ts

```diff
@@ -100,6 +101,21 @@ export class MessageService
                 this.cooldownService.addCooldown(message.member, "XP", 60);
         }
     }
+    
+    private async addToTotalPings(user: User, message:Message)
```

Comment:
`user: User` seems pretty obsolete here since you dont use it in that method

Reply:
An error comes up if i dont have 2 arguments